src/user/submit_workflow.rst

```diff
@@ -0,0 +1,227 @@
+  issue on Quantum Graph directory if a user is using the default configuration (without
+  overwritting **fileDistributionEndPoint**).
+
+
+Setup environment
+-----------------
+
+Setup environment with CVMFS
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The submit workflows from USDF, FrDF and UKDF, the CVMFS is available. Users can use
+CVMFS to setup the environments::
+
+  latest=$(ls -td /cvmfs/sw.lsst.eu/linux-x86_64/panda_env/v* | head -1)
+  source $latest/setup_lsst.sh <lsst stack version>
+  # source $latest/setup_lsst.sh w_2024_14   # for example
+  source $latest/setup_panda.sh
+
+For LSST stack verion older than w_2024_14, this line below is required for remote submission::
+
+  source $latest/setup_bps.sh
+
+Setup environment on IDF
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Rubin Science Platform (RSP) can be accessed from the JupyterLab
+notebook configured for the IDF at: ::
+
+    https://data-int.lsst.cloud/
+
+Choose "Notebooks" and authorize lsst-sqre with your user credentials.
+After successful authentication, choose a cached image or the latest weekly
+version (recommended) from the drop down menu.
+
+.. image:: ../_images/JupyterLab.png
+   :width: 6.5in
+   :height: 2.66667in
+
+Open a terminal (menu **File > New > Terminal**).
+
+On IDF, CVMFS is not available. However, on IDF, normal environments should already be setup.
+Users only need to run this command below after login to data-int.lsst.cloud or data.lsst.cloud. ::
+
+    setup lsst_distrib
+
+**IDF can only run remote submission currently, to submit workflows to USDF, FrDF or UKDF.**
+
+Ping PanDA Service
+~~~~~~~~~~~~~~~~~~
+
+After setting up the environment, users can try to ping PanDA service to make sure the settings are ok.
+
+If the BPS_WMS_SERVICE_CLASS is not set, set it through::
+
+   $> export BPS_WMS_SERVICE_CLASS=lsst.ctrl.bps.panda.PanDAService
+
+Ping the PanDA system to check whether the service is ok::
+
+   $> bps ping
+
+**In this step, it will look for tokens for authorization. If you don't have a token or you are not registered yet,
+please check** :ref:`user_authentication`.
+
+Prepare YAML configuration
+--------------------------
+
+YAML configuration
+~~~~~~~~~~~~~~~~~~
+
+As any other Rubin workflow submitted with BPS commands, PanDA based
+data processing requires a YAML configuration file. The YAML settings,
+common for different BPS plugins provided here::
+
+    https://pipelines.lsst.io/modules/lsst.ctrl.bps/quickstart.html#defining-a-submission
+
+Later in this section we focus on PanDA specific and minimal set of the
+common settings supplied in the YAML with *bps submit <config>.yaml*
+command. They are::
+
+   -  maxwalltime: 90000 maximum wall time on the execution node allowed to
+      run a single job in seconds
+
+   -  maxattempt: 1 number of attempts to successfully execute a job. It is
+      recommended to set this parameter at least to 5 due to preemptions
+      of machines used in the GKE cluster
```

Comment:
Is numberOfRetries (from core bps yaml) not used?

Reply:
The maxwalltime should be 'request_walltime'. The 'maxattempt' should be 'number_of_retries'. I fixed it.